Dockerfile

```diff
@@ -18,10 +18,15 @@ RUN cat rust-toolchain.toml | grep "channel" | awk '{print $3}' | sed 's/\"//g'
     && cargo install cargo-chef
 
 FROM chef AS planner
+ARG BUILD_DEBUG=false
```

Comment:
Why not to just proxy `CARGO_PROFILE_RELEASE_DEBUG` env var as-is? So we can set it as a build arg explicitly: `CARGO_PROFILE_RELEASE_DEBUG=true`?

Reply:
I don't want to tie it to much to how it's currently built. Ultimately I'l like to either build with split debug info, and discard them when not in debug for local builds, and build the image from the official build for official release, so that we can use the debug info in production as well